server/models/Analysis.js

```diff
@@ -0,0 +1,28 @@
+const keystone = require('keystone');
+const { Types } = keystone.Field;
+
+const Analysis = new keystone.List('Analysis');
+
+Analysis.add({
+    firstText: {
+        type: Types.Text,
+        required: true,
+        initial: true,
+        label: "Descrição no primeiro parágrafo"
+    },
+    secondText: {
+        type: Types.Text,
+        required: true,
+        initial: true,
+        label: "Descrição no segundo parágrafo"
+    },
+    firstImage: {
+        type: Types.CloudinaryImages,
+        require: true,
+    },
+    secondImage: {
+        type: Types.CloudinaryImages,
+        require: true
+    }
+})
```

Comment:
Well done, Carol!! All steps of the modular delivery back-end flow are correct. I have only one fix, I think it will be very complicated to create the logic in the front-end to separate the text of each paragraph into the exact topics in each paragraph, so an improvement would be to create a model structure where each paragraph could be given a maximum of three spaces that would be topics.

Reply:
Oh i get your point! But could you say more exactly what you mean by "each paragraph could be given a maximum of three spaces"? Would it be to put just one text in the back-end and divide it into a maximum of three topics?

Reply:
Oh, sorry if my explanation wasn't very good, but what I meant is that you could do something like this:
```js
Analysis.add({
 firstText: {
  firstTopic: {
   type: Types.Text,
   required: true,
   initial: true,
   max: 30,
   label: "Descrição no primeiro tópico" 
  },
  secondTopic...
})
```

That way you would have defined each topic, and also defined a maximum number of characters so that no layout errors occur in the front end. Don't worry about defining exactly the maximum number of characters for each topic now, it will be defined during the development of the front end.

Reply:
Sorry, I saw your message and thought I had replied it. I'll make the changes as soon as possible. Just another question: should I merge the branch before these changes? Because the branch is not in my computer yet :(